lib/rules/eol-last.js

```diff
@@ -1,62 +1,95 @@
 /**
- * @fileoverview Require file to end with single newline.
+ * @fileoverview Require or disallow file to end with single newline.
  * @author Nodeca Team <https://github.com/nodeca>
+ * @author kdex <https://github.com/kdex>
```

Comment:
Sorry! I'll be removing mine. Note that I have seen cases where additional authors were added [years](https://github.com/eslint/eslint/blame/4126f12f792231c844859e13a3fbd6b7ff78c4ee/lib/rules/no-plusplus.js) after the file has been created, up to [this year](https://github.com/eslint/eslint/blame/c67632259783c9630831bce5f272eb6160ee086c/lib/rules/no-duplicate-case.js), which I followed. Did these cases slip under the radar?